scripts.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+function isJSON(string) {
+  try {
+    JSON.parse(string);
+    return true;
+  }
+  catch {
```

Comment:
There is a more convenient way to write catch on the same line with try close bracket(same as if-else)